modules/preview-react/form-field/lib/FormFieldLabel.tsx

```diff
@@ -25,7 +25,7 @@ const asteriskStyles = createStyles({
   fontSize: type.properties.fontSizes[20],
   fontWeight: type.properties.fontWeights.regular,
   textDecoration: 'unset',
-  color: cssVar(brand.error.base, '#de2e21'),
+  color: brand.error.base,
```

Comment:
This type of change is in many places. Auto-fallback works and manually including a fallback is a maitenance issue.